docker-compose.yml

```diff
@@ -26,6 +27,24 @@ services:
     image: elasticsearch:2
     restart: always
 
+  rabbitmq:
```

Comment:
What will RabbitMQ be adding to the system? It seems like we have a lot of dependencies that we are expecting our customer to run. Will follow up in slack.